frontend/src/common/mock/common/data.ts

```diff
@@ -8,3 +8,10 @@ export const SPECIALTIES: Specialty[] = [
   { id: '5', title: '영양 상담' },
   { id: '6', title: '재활 운동' },
 ] as const;
+
+export const USER_INFO = {
+  loginId: 1,
+  name: '홍길동',
+  gender: '남',
+  phone: '010-1234-5678',
+};
```

Comment:
p4: api 명세에 imageUrl 이 추가되어서 이 데이터에도 imageUrl을 추가하면 될 거 같아!

그리고 mockData의 변화를 감지하기 쉽게 여기에 타입을 지정해놓는 건 어떨까? 🤔 

타입 설정을 해두면 api명세가 빠르게 바뀔 때 추적하기 쉬울 거 같아 ㅎㅎ

```suggestion
interface UserInfo {
    "image": string,
    "loginId": number,
    "name": string,
    "gender": "남" | "여",
    "phone": string,
}

export const USER_INFO:UserInfo[] = {
```

Reply:
타입 변경 추적하기 좋을 거 같네!! 굿굿~

Reply:
[refactor: 내정보 조회 api를 위한 UserInfo mockData 변경 및 UserInfo 타입 변경](https://github.com/woowacourse-teams/2025-Fit-toring/pull/289/commits/47bb92752eecd344fa379a8b986d82c3f4896386)